packages/api/src/services/lsp-state.ts

```diff
@@ -0,0 +1,129 @@
+import * as uma from "@uma/sdk";
+import { BatchRead, toString, toNumber, parseBytes, nowS } from "../libs/utils";
+import { AppState } from "..";
+import { lsps } from "../tables";
+
+type Instance = uma.clients.lsp.Instance;
+type Config = undefined;
+type Dependencies = Pick<
+  AppState,
+  "lsps" | "registeredLsps" | "provider" | "collateralAddresses" | "shortAddresses" | "longAddresses" | "multicall"
+>;
+
+export default (config: Config, appState: Dependencies) => {
+  const { lsps, registeredLsps, provider, collateralAddresses, shortAddresses, longAddresses, multicall } = appState;
+
+  // default props we want to query on contract
+  const staticProps: [string, (x: any) => any][] = [
+    // TODO: deal with this when early contracts dont have pairName
```

Comment:
I think there will only ever be one contract that does not have `pairName` which is the current mainnet range bond. all future ones will have this prop

Reply:
ok cool. I was scoping out the lsp creator to see if other contracts have launched, but didnt see any others. I think it will be safe to enable this, but it was causing the e2e tests to fail currently.